documentation/migration-guides/Az.3.0.0-migration-guide.md

```diff
@@ -0,0 +1,924 @@
+### `Disable-AzHDInsightMonitoring`
+Added a new `Disable-AzHDInsightMonitoring` cmdlet. Use this cmdlet to disable monitoring in a HDInsight cluster (replaces `Disable-AzHDInsightOperationsManagementSuite` and `Disable-AzHDInsightOMS`).
+
+#### Before
+```powershell
+Disable-AzHDInsightOMS -Name testcluster
+```
+```powershell
+Disable-AzHDInsightOperationsManagementSuite -Name testcluster
+```
+
+#### After
+```powershell
+Disable-AzHDInsightMonitoring -Name testcluster
+```
+
+
+### `Enable-AzHDInsightMonitoring`
+Added a new `Enable-AzHDInsightMonitoring` cmdlet. Use this cmdlet to enable monitoring in a HDInsight cluster (replaces `Enable-AzHDInsightOperationsManagementSuite` and `Enable-AzHDInsightOMS`).
+
+#### Before
+```powershell
+Enable-AzHDInsightOMS Enable-AzHDInsightMonitoring -Name testcluster -WorkspaceId 1d364e89-bb71-4503-aa3d-a23535aea7bd -PrimaryKey <key for workspace 1d364e89-bb71-4503-aa3d-a23535aea7bd>
+```
+```powershell
+Enable-AzHDInsightOperationsManagementSuite Enable-AzHDInsightMonitoring -Name testcluster -WorkspaceId 1d364e89-bb71-4503-aa3d-a23535aea7bd -PrimaryKey <key for workspace 1d364e89-bb71-4503-aa3d-a23535aea7bd>
+```
+
+#### After
+```powershell
+Enable-AzHDInsightMonitoring Enable-AzHDInsightMonitoring -Name testcluster -WorkspaceId 1d364e89-bb71-4503-aa3d-a23535aea7bd -PrimaryKey <key for workspace 1d364e89-bb71-4503-aa3d-a23535aea7bd>
+```
+
+### `Get-AzHDInsightMonitoring`
+Added a new `Get-AzHDInsightMonitoring` cmdlet. Use this cmdlet to get the status of monitoring installation in an Azure HDInsight cluster (replaces `Get-AzHDInsightOperationsManagementSuite` and `Get-AzHDInsightOMS`).
+
+#### Before
+```powershell
+Get-AzHDInsightOMS -Name testcluster
+```
+```powershell
+Get-AzHDInsightOperationsManagementSuite -Name testcluster
+```
+
+#### After
+```powershell
+Get-AzHDInsightMonitoring -Name testcluster
+```
+
+### `Get-HDInsightProperty`
+Cmdlet `Get-HDInsightProperty` removed alias to `Get-AzHDInsightProperties`.
+
+#### Before
+Using deprecated alias
+```powershell
+Get-AzHDInsightProperties -Location "East US 2"
+```
+
+#### After
+```powershell
+Get-HDInsightProperty -Location "East US 2"
+```
+
+### `Grant-AzHDInsightRdpServicesAccess`
+Removed the `Grant-AzHDInsightRdpServicesAccess` and `Revoke-AzHDInsightRdpServicesAccess` cmdlets. These are no longer necessary because clusters using Windows OS type are not supported. Please create a cluster using Linux OS type instead.
+
+### `Remove-AzHDInsightCluster`
+The output type of `Remove-AzHDInsightCluster` changed from `Microsoft.Azure.Management.HDInsight.Models.ClusterGetResponse` to `bool`.
+
+#### Before
+```powershell
+$cluster = Remove-AzHDInsightCluster -ClusterName "your-hadoop-001"
+```
+
+#### After
+```powershell
+Remove-AzHDInsightCluster -ClusterName "your-hadoop-001" -PassThru
+True
+```
+
+### `Revoke-AzHDInsightRdpServicesAccess`
+The cmdlet is deprecated. There is no replacement for it.
+
+### `AzHDInsightGatewayCredential`
```

Comment:
```suggestion
### `Set-AzHDInsightGatewayCredential`
```

Reply:
We should add more detail here (can be later this week).  What PowerShell users will be interested in is how the properties have changed,  What properties of the old object are no longer available in the new type, and how can I get equivalents from the new type?